components/api_server/src/routes/logo.py

```diff
@@ -1,11 +1,16 @@
 """Source logos."""
 
 import bottle
+from bottle import abort, response
```

Comment:
When also importing `get` then the line above can be dropped

Reply:
Done.